sql/messaging_system_derived/whats_new_panel_users_daily_v1/query.sql

```diff
@@ -0,0 +1,51 @@
+WITH windowed AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    impression_id,
+    client_id,
+    ROW_NUMBER() OVER w1_unframed AS _n,
+    --
+    -- For all dimensions, we use the mode of observed values in the day.
+    udf.mode_last(ARRAY_AGG(release_channel) OVER w1) AS release_channel,
+    udf.mode_last(ARRAY_AGG(locale) OVER w1) AS locale,
+    udf.mode_last(ARRAY_AGG(metadata.geo.country) OVER w1) AS country,
+    udf.mode_last(ARRAY_AGG(version) OVER w1) AS version
+  FROM
+    messaging_system.cfr
+  WHERE
+    -- Reprocess all dates by running this query with --parameter=submission_date:DATE:NULL
+    (@submission_date IS NULL OR @submission_date = DATE(submission_timestamp))
+    -- Whats-new-panel pings are from `messaging_system.cfr` with a name pattern `%WHATS_NEW%`
+    -- in the `bucket_id` column
+    AND bucket_id LIKE '%WHATS_NEW%'
```

Comment:
We could actually roll this into the cfr_users_last_seen table as an additional usage criterion: `days_seen_whats_new_bits`.